pydrive2/files.py

```diff
@@ -97,6 +98,48 @@ def _GetList(self):
         return result
 
 
+class FakeWriteable(object):
+    def write(self, chunk):
+        self.chunk = chunk
+
+
+class MediaIoReadable(object):
+    def __init__(self, request, chunksize=DEFAULT_CHUNK_SIZE, pre_buffer=True):
```

Comment:
`pre_buffer` - feels like we adding it ahead of time?

Reply:
I see why we need this .. to make the right request type

Reply:
yes, unfortuantely